docs/source/getting-started/quickstart.rst

```diff
@@ -178,5 +196,5 @@ Next steps:
 - Adapt :ref:`Tutorial: DNN Training <dnn-training>` to start running your own project on SkyPilot!
 - See the :ref:`Task YAML reference <yaml-spec>`, :ref:`CLI reference <cli>`, and `more examples <https://github.com/skypilot-org/skypilot/tree/master/examples>`_
 - To learn more, try out `SkyPilot Tutorials <https://github.com/skypilot-org/skypilot-tutorial>`_ in Jupyter notebooks
-- Try :ref:`Interactive Nodes <interactive-nodes>` -- launch VMs in one command without a YAML file
-- Explore SkyPilot's unique features in the rest of the documentation
+
+We invite you to explore SkyPilot's unique features in the rest of the documentation.
```

Comment:
Maybe we can leave this out, less fluff

Reply:
This seems like a reasonable tone to me. Points out "there are more unique features".